### ISSUE TITLE
[Bugs] 1.5 RC1 Geeklog - using a % in the Mail Settings[username]  cause error

### ISSUE THREAD

**Issue by mystralkk** [Geeklog-Core/geeklog#140]
**Reported by furono on 29 May 2008 06:34**

**Version:** 1.5.0

[Mantis Bugtracker #649](http://project.geeklog.net/tracking/view.php?id=649)

**Comment by mystralkk** [comment]
**Comment posted by dhaun on 29 May 2008 18:59**

Works for me.

Server version: 5.0.45 MySQL Community Server (GPL)
(...)
mysql&gt; select value from sp_conf_values where name = &#039;mail_settings&#039;;
(...)
| a:8:{s:7:&quot;backend&quot;;s:4:&quot;smtp&quot;;s:13:&quot;sendmail_path&quot;;s:17:&quot;/usr/bin/sendmail&quot;;s:13:&quot;sendmail_args&quot;;s:0:&quot;&quot;;s:4:&quot;host&quot;;s:16:&quot;smtp.tinyweb.net&quot;;s:4:&quot;port&quot;;i:25;s:4:&quot;auth&quot;;i:1;s:8:&quot;username&quot;;s:16:&quot;bbs%gbless.co.jp&quot;;s:8:&quot;password&quot;;s:8:&quot;password&quot;;} |

Aaron, any comments?

**Comment by mystralkk** [comment]
**Comment posted by furono on 30 May 2008 02:20**

I appologize , I think I figured out most of it, its maybe a modsecurity problem however I&#039;m not having this issue with anything else I use - ie : Geeklog older versions, any other php mysql web application, various Java mysql apps, plus a few other web mysql applications - all running off the same server.

After using Geeklog config to set it Modsecurity is catching it and saying &quot;URL Encoding Abuse Attack Attempt&quot; and gives the above response.  

I was able to directly insert it into the database after I figured out what the values meant in mail_settings.  Sorry for saying it didnt work.  I most likely can tweek something in modsecurity but would rather not.

I get the same response on a clean install or upgrade.  I&#039;m using UTF-8 for everything, Geeklog, PHP Mysql.

Again sorry should of investigated it more.

Thanks
